src/components/organisms/ProblemSection.jsx

```diff
@@ -32,12 +32,15 @@ function ProblemSection () {
                  The Problem
                 </Typography>
                 <Typography variant="body1" style={{fontWeight: "300", lineHeight: "1.4"}}>
-                  Math, ugh! Many students feel that math is all about tests,
+                  {/* Math, ugh! Many students feel that math is all about tests,
                   grades, homework, and getting correct answers... What if we
                   could flip things around and make math more about learning
                   from mistakes and even having fun? Give
                   {" "}<b><em>RightOn!</em></b>{" "}
-                  a try and find out how!
+                  a try and find out how! */}
```

Comment:
Why don't we go ahead and delete this text entirely. Because we use git, we'll always have the old text in the commit history. 😄

Reply:
Haha yeah, I forgot about the comments to be honest. Will do!